services/security/identity/identity.go

```diff
@@ -113,6 +110,14 @@ func getWssdIdentity(id *security.Identity) (*wssdcloudsecurity.Identity, error)
 
 	wssdidentity.ClientType = clitype
 	wssdidentity.AutoRotate = id.AutoRotate
+
+	// In case auto rotate in enabled, the login file path can not be relative
+	if id.AutoRotate == true && id.LoginFilePath != nil {
+		if !(filepath.IsAbs(*id.LoginFilePath)) {
```

Comment:
This will throw an error when the LoginFilePath is empty, leading to the same issue

Reply:
Thanks, sorry that I missed this case. I have added a condition so that it will only check whether a path is absolute if
1. The auto rotate flag is enabled
2. The login file path is not nil
3. Login file path is not empty

Reply:
Can we write a few units test so that we don't regress in future